expression/schema.go

```diff
@@ -233,3 +234,24 @@ func MergeSchema(lSchema, rSchema *Schema) *Schema {
 func NewSchema(cols ...*Column) *Schema {
 	return &Schema{Columns: cols}
 }
+
+// NamingForMySQLProtocol records the names used for mysql protocol.
+type NamingForMySQLProtocol struct {
+	OrigTblName model.CIStr
+	OrigColName model.CIStr
+	DBName      model.CIStr
+	TblName     model.CIStr
+	ColName     model.CIStr
+}
+
+// String implements Stringer interface.
+func (name *NamingForMySQLProtocol) String() string {
+	result := name.ColName.L
```

Comment:
It is better to put backquote aroud names.

Reply:
It needs to keep the same with `Column.String()`, so no backquote currently.